pkg/cedar/intake/cedar_intake.yml

```diff
@@ -126,42 +126,7 @@ definitions:
       - asIsPros
       - asIsCons
       - asIsCostSavings
-      - preferredTitle
-      - preferredSummary
-      - preferredAcquisitionApproach
-      - preferredSecurityIsApproved
-      - preferredSecurityIsBeingReviewed
-      - preferredHostingType
-      - preferredHostingLocation
-      - preferredHostingCloudServiceType
-      - preferredHasUI
-      - preferredPros
-      - preferredCons
-      - preferredCostSavings
-      - alternativeATitle
-      - alternativeASummary
-      - alternativeAAcquisitionApproach
-      - alternativeASecurityIsApproved
-      - alternativeASecurityIsBeingReviewed
-      - alternativeAHostingType
-      - alternativeAHostingLocation
-      - alternativeAHostingCloudServiceType
-      - alternativeAHasUI
-      - alternativeAPros
-      - alternativeACons
-      - alternativeACostSavings
-      - alternativeBTitle
-      - alternativeBSummary
-      - alternativeBAcquisitionApproach
-      - alternativeBSecurityIsApproved
-      - alternativeBSecurityIsBeingReviewed
-      - alternativeBHostingType
-      - alternativeBHostingLocation
-      - alternativeBHostingCloudServiceType
-      - alternativeBHasUI
-      - alternativeBPros
-      - alternativeBCons
-      - alternativeBCostSavings
+      - businessSolutions
```

Comment:
Should the As-Is solution be included among the businessSolutions?

Reply:
I thought about this but because it doesn't have all the same fields as preferred and the alternatives. It doesn't ask for security or hosting for as is.

I am fine with changing this to have it but then we would be sending empty fields and I'm not sure if there's precedent for doing that with our CEDAR connection.